bibblio.py

```diff
@@ -312,7 +463,12 @@ def get_full_text(self, work):
             .join(LicensePoolDeliveryMechanism.delivery_mechanism)\
             .filter(
                 LicensePool.work_id==work.id,
-                DeliveryMechanism.drm_scheme==DeliveryMechanism.NO_DRM)\
+                DeliveryMechanism.drm_scheme==DeliveryMechanism.NO_DRM,
+                or_(
+                    Representation.status_code==200,
+                    Representation.status_code==None
+                )
+            )\
```

Comment:
I tried this and ran into errors with the `exists()` clause in that method not auto-correlated with the required classes -- possibly because this queries Representation instead of Work? Or because LicensePool and LicensePoolDeliveryMechanism had been joined in unexpected ways? I'm not sure, but it didn't seem worth it to keep fighting with it.